SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -4503,16 +4506,226 @@
+      "OutputDtmi": "dtmi:com:willowinc:OutsideAirFlowSetpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Outside_Air_Humidity_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:OutsideAirHumiditySensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Outside_Air_Temperature_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:OutsideAirTemperatureSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Power_Factor_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:PowerFactorSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Relative_Humidity_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:RelativeHumiditySensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Reset_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:ResetSetpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Return_Air_Temperature_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:ReturnAirTemperatureSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Speed_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:SpeedSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Static_Pressure_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:StaticPressureSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Static_Pressure_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:StaticPressureSetpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Supply_Fan;1",
+      "OutputDtmi": "dtmi:com:willowinc:SupplyFan;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Temperature_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:TemperatureSetpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Torque_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:TorqueSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Valve_Position_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:ValvePositionSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Valve_Position_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:ValvePositionSetpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Voltage_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:VoltageSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Water_Temperature_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:WaterTemperatureSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Wind_Direction_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:WindDirectionSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Air_Temperature_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:ZoneAirTemperatureSetpoint;1"
+    },
+
+    // Mapped.Core
```

Comment:
There's a whole bunch of Mapped.Core things further up, should those be moved here? Or these merged up there?